ngraph/core/include/ngraph/op/ctc_greedy_decoder_seq_len.hpp

```diff
@@ -58,18 +58,26 @@ namespace ngraph
                 std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
 
+                /// \brief Get merge_repeated attribute
                 bool get_merge_repeated() const { return m_merge_repeated; }
+
+                /// \brief Get classes_index_type attribute
                 const element::Type& get_classes_index_type() const { return m_classes_index_type; }
+
+                /// \brief Set classes_index_type attribute
```

Comment:
Please add information about arguments and return types.
You can use this example as reference: https://github.com/openvinotoolkit/openvino/blob/84b5fc51dcb63f3f83e8ed8954d32bc979da4f71/ngraph/core/include/ngraph/op/util/sub_graph_base.hpp#L308